nexus-repository-p2-it/src/test/java/org/sonatype/nexus/repository/p2/internal/P2ProxyIT.java

```diff
@@ -45,7 +45,9 @@
 
   private static final String MIME_TYPE = "application/java-archive";
 
-  private static final String COMPONENT_NAME = "org.eclipse.cvs.source";
+  private static final String COMPONENT_NAME = "%featureName";
```

Comment:
Well, its a component with name '%featureName', because in feature.xml of org.eclipse.cvs.source.jar file we have record like <feature id="org.eclipse.cvs.source" version="1.4.404.v20180330-0640" label="%featureName" provider-name="%providerName">
In this case we should use a file feature.properties.
But I will fix it in next PRs as it is not root case, it is only one jar.